### PR TITLE
solving abnormal classfication loss by adding background class

### PR DESCRIPTION
**Pull request by HannH**
[#54](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/54)
[#58](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/58)

Must add background class when calculating class loss in https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/96570fbed3ae5c2a8772a91feb27685e4ce32bf9/efficientdet/loss.py#L94

Other implemention of Focal loss for reference, they add background class as well.
https://github.com/open-mmlab/mmdetection/blob/25ede58af6072ea8089af7b42cbb418a3b6991e9/configs/retinanet_r50_fpn_1x.py#L22

### CODE REVIEW

**Comment by zylo117** [comment]
No, this is wrong. Pretrained efficientdet doesn't have a backgrond class.

Is there a backgrond class in VOC dataset whose index is -1?

Then why don't you set background class index to the end of the class index, like 21?

Then no code needs to be changed.

**Comment by zylo117** [comment]
there's a bug in loss func, so sorry for the trouble. please pull the latest code and try again
